src/app/components/ClearAllButton/ClearAllButton.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+import styled from 'styled-components'
+
+type ClearAllButtonProps = {
+  children: ReactNode
+  onClick: () => void
+}
+
+export default function ClearAllButton({
+  children,
+  onClick,
+}: ClearAllButtonProps): JSX.Element {
+  return <Button onClick={onClick}>{children}</Button>
+}
+
+const Button = styled.button`
```

Comment:
If you are really bored, you can even sort the different css styles like: grid/size first, then everything with color, etc.